MN/Algoritmi/Metode-Iterative/SOR/SOR.m

```diff
@@ -0,0 +1,31 @@
+%  Solves linear systems A*x = B using super-over-relaxation method
+%  success - 0 = solution found / 1 = maxiter exceeded
+
+function [x success] = SOR(A, b, x0, w, tol, maxiter)
```

Comment:
cred ca o abordare [x step] e mai folositoare fiindca:
1) poate sa convearga la ultimul pas sau sa fie foarte aproape dupa maxiter pasi, nu inseamna ca e fail
2) daca merge foarte bine vedem in cati pasi a mers raportat la maxiter
3) putem compara mai usor alegerile lui w